src/main/java/com/synopsys/integration/detect/lifecycle/run/operation/OperationRunner.java

```diff
@@ -498,6 +498,20 @@ public void generateComponentLocationAnalysisIfEnabled(List<DeveloperScansScanVi
         }
     }
 
+    /**
+     * Since component location analysis is not supported for online Intelligent scans in 8.11, an appropriate console
+     * msg is logged and status=FAILURE is recorded in the status.json file
+     * @throws OperationException
+     */
+    public void attemptToGenerateComponentLocationAnalysisIfEnabled() throws OperationException {
+        if (detectConfigurationFactory.isComponentLocationAnalysisEnabled()) {
+            auditLog.namedPublic(
+                    "Generate Component Location Analysis File for All Components",
+                    () -> (new GenerateComponentLocationAnalysisOperation()).locateComponentsForOnlineIntelligentScan()
+            );
+        }
+    }
+
```

Comment:
The ask in IDETECT-3962 was to log an appropriate msg to console instead of silently ignoring this feature being enabled for an unsupported scan mode. Instead of a simple if statement in IntelligentModeStepRunner, I had to do it in this convoluted way because see: https://github.com/blackducksoftware/synopsys-detect/pull/844#discussion_r1262432125